phpBB/config/default/container/services_twig_extensions.yml

```diff
@@ -0,0 +1,12 @@
+# Twig extensions don't needed by the installer
+
+services:
+    template.twig.extensions.jumpbox:
+        class: phpbb\template\twig\extension\jumpbox
```

Comment:
Tlaking to @marc1706 we are going to be enforceing a naming convention for all of our twig funcitns that return template code using Pascal Case so please use JumpBox for the function name to avoid clshing with css class names